frontend/src/styles/VendorCard.css

```diff
@@ -23,13 +23,14 @@
 .vendorLogo {
   min-height: 100px;
   min-width: 100px;
+  max-height: 100px;
```

Comment:
Így már jó lesz, de ha valamit minimalizálsz X-re és maximalizálsz X-re, akkor be is állíthatnád fixen X-re.